diagnostic.md

```diff
@@ -5,74 +5,77 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The back end, or server side, is code that remains on the server.
 ```
```

Comment:
Think of it as what interacts with and accesses the database based on user requests.